sdk/eventhub/Azure.Messaging.EventHubs.Processor/tests/CheckpointStore/BlobsCheckpointStoreTests.cs

```diff
@@ -616,6 +616,42 @@ public async Task ListCheckpointsUsesSequenceNumberAsTheStartingPositionWhenNoOf
             Assert.That(checkpoints.Single().PartitionId, Is.EqualTo("0"));
         }
 
+        /// <summary>
+        ///   Verifies basic functionality of ListCheckpointsAsync and ensures the starting position is set correctly.
+        /// </summary>
+        ///
+        [Test]
+        public async Task ListCheckpointsUsesSequenceNumberAsTheStartingPositionWhenOffsetIsNullInLegacyCheckpoint()
+        {
+            var blobList = new List<BlobItem>
+            {
+                BlobsModelFactory.BlobItem($"{FullyQualifiedNamespace}/{EventHubName}/{ConsumerGroup}/0",
+                                           false,
+                                           BlobsModelFactory.BlobItemProperties(true, lastModified: DateTime.UtcNow, eTag: new ETag(MatchingEtag)),
+                                           "snapshot")
+            };
+
+            var containerClient = new MockBlobContainerClient() { Blobs = blobList };
+            containerClient.AddBlobClient($"{FullyQualifiedNamespace}/{EventHubName}/{ConsumerGroup}/0", client =>
+            {
+                client.Content = Encoding.UTF8.GetBytes("{" +
+                                                        "\"Offset\":null," +
```

Comment:
would it actually say "null" or just be blank?

Reply:
This is the actual checkpoint value that was failing